src/config/settings.json

```diff
@@ -0,0 +1,46 @@
+{

```

Comment:
This config file is imported into the engine lib binary, but it allows for a more flexible setup than incode configuration.